ui/ui-app/src/main/resources/static/js/feed-mgr/visual-query/transform-data/visual-query-table/visual-query-table.component.ts

```diff
@@ -158,7 +158,7 @@ export class VisualQueryTable {
            if(resizeTimeoutPromise != null){
                this.$timeout_.cancel(resizeTimeoutPromise);
            }
-           resizeTimeoutPromise = this.$timeout_(callback,interval);
+           resizeTimeoutPromise = this.$timeout_(callback(),interval);
```

Comment:
This looks like it's executing the callback immediately instead of waiting for the timeout to occur. Could you verify if that's the case?